src/foam/util/UIDGeneratorTest.js

```diff
@@ -22,6 +22,12 @@ foam.CLASS({
         var hash   = mod("foobar");
         UIDGeneratorTest_GenerateVerifiableUniqueStringIDs(uidgen, hash);
         UIDGeneratorTest_GenerateVerifiableUniqueLongIDs(uidgen, hash);
+        // UIDs similarity tests
```

Comment:
please move test to util/test/

Reply:
Fixed in https://github.com/kgrgreer/foam3/pull/1131/commits/182148f3b2e3e7a5fa1ff2e3673a868a044ee708.